stdlib/REPL/test/replcompletions.jl

```diff
@@ -537,27 +547,56 @@ let s = "CompletionFoo.kwtest( "
     @test !res
     @test length(c) == 1
     @test occursin("x, y, w...", c[1])
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(;")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(; x=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(; kw=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(x=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(x=1; ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(x=kw=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest(; x=kw=1, ")
 end
 
-for s in ("CompletionFoo.kwtest(;",
-          "CompletionFoo.kwtest(; x=1, ",
-          "CompletionFoo.kwtest(; kw=1, ",
-          )
+let s = "CompletionFoo.kwtest2(1, x=1,"
     c, r, res = test_complete(s)
     @test !res
     @test length(c) == 1
-    @test occursin("x, y, w...", c[1])
+    @test occursin("a; x, y, w...", c[1])
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(1; x=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(1, x=1; ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(1, kw=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(1; kw=1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(1, kw=1; ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(y=3, 1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(y=3, 1; ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(kw=3, 1, ")
+    @test (c, r, res) == test_complete("CompletionFoo.kwtest2(kw=3, 1; ")
```

Comment:
Why shouldn't these example be completed?
For example, `CompletionFoo.kwtest2(1; |` is better provide keyword completions?

Reply:
This is testing that the completion is the same for all these inputs, so it is indeed checking the these examples are completed. The completion is not empty since we're checking just above that `length(c) == 1`.
I added two new lines
```julia
@test (c, r, res) == test_complete("CompletionFoo.kwtest2(1; )
@test (c, r, res) == test_complete("CompletionFoo.kwtest2(1, )
```
since you're right, the completion should also be the same without any given kwarg.